src/functions/keycloak/update-user/handler.ts

```diff
@@ -0,0 +1,47 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import { CreateKeycloakApiService } from "src/database/services/keycloak-api-service";
+import { middyfy } from "src/libs/lambda";
+
+/**
+ * Lambda handler to update a user's attributes
+ * 
+ * @param event API Gateway event 
+ * @returns Response message as JSON string
+ */
+const updateUserAttributeHandler: APIGatewayProxyHandler = async (
+  event: APIGatewayProxyEvent
+) => {
+  try {
+    if (!event.body) {
+      throw new Error("Request body is missing.");
+    }
+
+    const body = JSON.parse(JSON.stringify(event.body));
+    const { id, attributes } = body;
```

Comment:
Why not attribute rather than attributes? If for now it's single the name should be singular.